phys2denoise.py

```diff
@@ -0,0 +1,404 @@
+
+                # If any filename exists already because of multirun, append labels
+                # But warn about the non-validity of this BIDS-like name.
+                if run_amount > 1:
+                    if any([phys.filename == phys_out[key].filename
+                           for phys in phys_out.values()]):
+                        phys_out[key].filename = (f'{phys_out[key].filename}'
+                                                  f'_take-{run}')
+                        LGR.warning('Identified multiple outputs with the same name.\n'
+                                    'Appending fake label to avoid overwriting.\n'
+                                    '!!! ATTENTION !!! the output is not BIDS compliant.\n'
+                                    'Please check heuristics to solve the problem.')
+
+            else:
+                phys_out[key].filename = os.path.join(outdir,
+                                                      os.path.splitext(os.path.basename(filename)
+                                                                       )[0])
+                # Append "run" to filename if more than one run
+                if run_amount > 1:
+                    phys_out[key].filename = f'{phys_out[key].filename}_{run:02d}'
+                # Append "freq" to filename if more than one freq
+                if freq_amount > 1:
+                    phys_out[key].filename = f'{phys_out[key].filename}_{uniq_freq:.0f}Hz'
+
+            LGR.info(f'Exporting files for run {run} freq {uniq_freq}')
+            np.savetxt(phys_out[key].filename + '.tsv.gz',
+                       phys_out[key].timeseries, fmt='%.8e', delimiter='\t')
+            print_json(phys_out[key].filename, phys_out[key].freq,
+                       phys_out[key].start_time, phys_out[key].ch_name)
+            print_summary(filename, num_timepoints_expected,
+                          phys_in[run].num_timepoints_found, uniq_freq,
+                          phys_out[key].start_time,
+                          os.path.join(conversion_path,
+                                       os.path.splitext(os.path.basename(phys_out[key].filename)
+                                                        )[0]))
+
+
+def _main(argv=None):
+    options = _get_parser().parse_args(argv)
+    phys2bids(**vars(options))
+
+
+if __name__ == '__main__':
+    _main(sys.argv[1:])
+
+"""
+Copyright 2019, The Phys2BIDS community.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
```

Comment:
I'm just curious- why is this in the actual file? Doesn't the LICENSE file at the top level apply to all files in the repository?